event_session/models/event.py

```diff
@@ -39,6 +39,9 @@ class EventEvent(models.Model):
         string=" # No of Confirmed Registrations",
         store=True,
     )
+    generate_sessions_button_visible = fields.Boolean(
+        compute="_compute_generate_sessions_button_visible",
+    )
```

Comment:
What about to have a relate field (`stage_id.pipe_end`) instead?

Reply:
Ok, I like it :)

Reply:
I like it